### DISCUSSION TITLE
Any idea what happends if cpu exceeds 100% in production?

### DISCUSSION THREAD

**Discussion by astrocket** [docker-mailserver/docker-mailserver#2534] [Q&A] [answered]
I'm using a fixed size instance with limited cpu.
Anyone know what happens with docker-mailser container if cpu exceeds the size of it's host?
In my knowledge I think it should just suppress and slow down, but wondering if there is a chance to kill itself?

**Comment by casperklein** [comment, marked as answer]
> In my knowledge I think it should just suppress and slow down,

I think so too. Let us know, what you experience.

Running out of memory (OOM) on the other hand is more critical. This might indeed kill your DMS instance. So ensure, there is enough memory or disable ClamAV e.g., which is known to consume a lot of RAM.

**Comment by georglauterbach** [comment]
With Docker and containerd, CPU is fine, but as @casperklein mentioned, running out of memory will result in the OOM killer giving you a hard time.

When using Kubernetes, it depends on the container runtime used. Some will actually kill the process when it exceeds a certain, but user-defined CPU limit.
